AUDIO/HintMissionAudioBank.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: AUDIO
+aliases: ["0x40763EA7B9B783E7"]
+---
+## HINT_MISSION_AUDIO_BANK
+
+```c
+// 0x40763EA7B9B783E7
+cs_type(Any) BOOL HINT_MISSION_AUDIO_BANK(char* bankName, cs_type(int) BOOL bOverNetwork, int playerBits);
+```
+
+This is marked as a deprecated function internally, please use [HINT_SCRIPT_AUDIO_BANK](#_0xFB380A29641EC31A) instead.
```

Comment:
Where did you get this? "marked as a deprecated function internally".

This native works correctly, it just dont guarentee the loading of the bank name, that's why `HINT_SCRIPT_AUDIO_BANK` is a better use.

Reply:
That is how the natives are marked in the docs.